docs/Bios.md

```diff
@@ -0,0 +1,8 @@
+# Introduction
+
+[Milin Joshi's online portfolio](http://milincjoshi.weebly.com/)
+
+Attribute  | Value
```

Comment:
This markdown formatting is easy and awesome. Still you should add your favorite color.